docs/REST API/rest-api.md

```diff
@@ -8930,6 +8933,7 @@ The script will be added to each host's list of upcoming activities.
 | script_id       | integer | body | **Required**. The ID of the existing saved script to run. |
 | host_ids        | array   | body |  List of host IDs.  Required if `filters` not specified. Only one of `host_ids` or `filters` may be included in the request.   |                                            |
 | filters | object  | body | See [filters](#filters3). Required if `host_ids` not specified. Only one of `host_ids` or `filters` may be included in the request.   |
+| starts_at       | string  | body | UTC time when the script run is scheduled for. |
```

Comment:
`not_before` would be a more accurate / standard field name for this.  We can't guarantee it'll start at the specified time, but we can say it won't run _until_ that time.  We can also add `not_after` later to allow for scheduled jobs to expire.

Reply:
👍 I like it.
```suggestion
| not_before       | string  | body | UTC time when the script run is scheduled to begin. |
```